### DISCUSSION TITLE
what is the cost/loss function in the machine learning?

### DISCUSSION THREAD

**Discussion by codeperfectplus** [Py-Contributors/support#399] [Q&A]
In simple words, the cost or the lost function, is the equation to find out how much error between predicted and actual output, it's used in machine learning, using backpropagation to update the model's weight to make the equation better fit on the data.
